tests/postgres_connector.rs

```diff
@@ -0,0 +1,97 @@
+use postgres::Config;
```

Comment:
You should write the tests to the postgres connector.

Reply:
Aka I should put them in the same file?

Reply:
No, write the tests to the code file you're testing. In this case, after the code, add

```rust
#[cfg(test)]
mod tests {
...
}
```

Reply:
Could we postpone that after prisma day? Right now we have three variants in prisma: test folder, doc test, test in the same file. 

Let's set this when we have more time. :)